features/api/applications/vertical_nav.feature

```diff
@@ -0,0 +1,10 @@
+  Feature: Vertical nav
```

Comment:
Vertical nav behavior is tested already under features/old/menu and it organized by context (Audience, Dashboard, etc).
I would suggest moving everything from features/old/menu -> features/menu and re-write the scenarios following the article ["Gherkin good practices"](https://cucumber.io/docs/bdd/better-gherkin/).

The general idea is to write "When bob does this, that happens" rather than "When I do this and that, I should see this thing".